### ISSUE TITLE
[Bug]  userChrome.css is not enforced.

### ISSUE THREAD

**Issue by lelanthran** [Floorp-Projects/Floorp#536]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

I've modified the userChrome.css file in the profiles directory with the attached content (see additional content field). But it does not take effect when I visit the page `https://news.ycombinator.com/item?id=38025014`.

### To Reproduce

1. Click on the 3 vertical dots menu.
2. Select "Open Profile Directory".
3. In the file browser that opens up, enter the directory "chrome".
4. Edit the file `userChrome.css` and add in the content in the additional field of this report.
5. Save and close the file.
6. Restart Floorp.
7. Navigate to [this page](https://news.ycombinator.com/item?id=38025014)
8. Note that none of the additional css settings in `userChrome.css` have been applied (max-width, font-size, etc for class `comment`, etc).
9. 

### Expected behavior

I expect the snippets I added to userChrome.css to take effect for the domain specified (news.ycombinator.com)

### OS Info

Windows 11 Home Single Language 22H2

### Floorp Version

11.5.0 (64-bit)

### Additional context

```
/*************************************************************************************************************************************************************************************************************************************************************

"userChrome.css" is a custom CSS file that can be used to specify CSS style rules for Floorp's interface (NOT internal site) using "chrome" privileges.
For instance, if you want to hide the tab bar, you can use the following CSS rule:

**************************************
#TabsToolbar {                       *
    display: none !important;        *
}                                    *
**************************************

NOTE: You can use the userChrome.css file without change preferences (about:config)

Quote: https://userChrome.org | https://github.com/topics/userchrome 

************************************************************************************************************************************************************************************************************************************************************/

@charset "UTF-8";
@-moz-document url(chrome://browser/content/browser.xhtml) {
/* Please write your custom CSS under this line*/

}

@-moz-document domain(news.ycombinator.com) {
   .comment {
      font-size: 12px;
      line-height: 1.75em;
      padding: 10px 0 10px 0;
      max-width: 64ch;
      display: block;
   }
   .commtext {
      font-size: 12px;
      line-height: 1.75em;
      padding: 10px 0 10px 0;
      max-width: 64ch;
      display: block;
   }
}
```

**Comment by menndouyukkuri** [comment]
userChrome.css is CSS that changes the look and feel of Floorp (Firefox). If you want to change the look and feel of the website, you must use userContent.css.

**Comment by lelanthran** [comment]
> userChrome.css is CSS that changes the look and feel of Floorp (Firefox). If you want to change the look and feel of the website, you must use userContent.css.

Thank you. I had the exact same content in both files, but it was not taking effect. I tracked it down to an error in the comment of userContent.css file that was shipped: it had recursive comments.

I removed the erroneous comment (recursive comments are apparently not supported in .css files).

I'm posting my solution in the hopes that anyone googling for the same problem finds it.
